ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -499,6 +525,14 @@ export default defineComponent({
     background-color: $gray1 !important;
   }
 
+  .no-bottom-border {
+    border-bottom: none !important;
+  }
+
+  .death-certificate {
```

Comment:
`death-certificate-row` might be a bit more accurate. 
The root component usually has an id that would be called the name of the component.